app/controllers/comments_controller.rb

```diff
@@ -1,6 +1,7 @@
 class CommentsController < ApplicationController
   before_action :set_comment, only: %i[ show edit update destroy ]
   before_action :is_an_authorized_user, only: [:destroy, :create]
```

Comment:
Don't need this anymore since we want to use Pundit

Reply:
Done!